crux-http-server/src/crux/ui/events.cljs

```diff
@@ -0,0 +1,113 @@
+(ns crux.ui.events
+  (:require
+   [ajax.edn :as ajax-edn]
+   [cljs.reader :refer [read-string]]
+   [clojure.string :as string]
+   [day8.re-frame.http-fx]
+   [re-frame.core :as rf]
+   [tick.alpha.api :as t]))
+
+(rf/reg-event-fx
+ ::inject-metadata
+ (fn [{:keys [db]} [_ title handler]]
+   (let [result-meta (some-> (js/document.querySelector
+                              (str "meta[title=" title "]"))
+                             (.getAttribute "content"))
+         edn-content (read-string result-meta)]
+     (if edn-content
+       {:db (assoc db handler edn-content)}
+       (js/console.warn "Metadata not found")))))
+
+(defn cast-to-query-params
+  [query valid-time transaction-time]
+  (let [{:keys [find where limit
+                offset args order-by]} query]
+    (cond-> ""
```

Comment:
can we use something that'll deal with url-encoding these parameters? `URLSearchParams`, I think you've used elsewhere?